vck/src/commonMain/kotlin/at/asitplus/wallet/lib/data/rfc7519/Rfc7519JwtHeaderParameterProvider.kt

```diff
@@ -0,0 +1,9 @@
+package at.asitplus.wallet.lib.data.rfc7519
+
+import at.asitplus.wallet.lib.data.rfc7519.Rfc7519JwtHeaderParameterProvider
+import at.asitplus.wallet.lib.data.rfc7519.jwt.headers.JwtContentTypeHeaderParameterSpecification
+import at.asitplus.wallet.lib.data.rfc7519.jwt.headers.JwtTypeHeaderParameterSpecification
+
+interface Rfc7519JwtHeaderParameterProvider : JwtContentTypeHeaderParameterSpecification.ParameterProvider,
```

Comment:
This interface is essentially never used, and also implements itself?!

Reply:
removed